R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -1173,7 +1173,7 @@ test_that("group by, agg functions", {
 
   expect_equal(3, count(mean(gd)))
   expect_equal(3, count(max(gd)))
-  expect_equal(30, collect(max(gd))[1, 2])
+  expect_equal(30, collect(max(gd))[2, 3])
```

Comment:
because the hash function in shuffle is changed in this PR, so the result of aggregate may have different order now.